drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2014,8 +2020,8 @@ int bnxt_hwrm_allocate_vfs(struct bnxt *bp, int num_vfs)
 		rc = -ENOMEM;
 		goto error_free;
 	}
-	for (ui = 0; ui < req_buf_sz; ui += rte_eal_get_physmem_size()) {
-		rte_mem_lock_page(((char *)bp->pf.vf_req_buf) + ui);
+	for (sz = 0; sz < req_buf_sz; sz += getpagesize()) {
```

Comment:
This is a bug fix to use proper pagesize. right?
Just want to make sure. So that the patch can be split into CP ring processing, cleanup and other fix(es).